qiime2/core/testing/method.py

```diff
@@ -117,7 +117,7 @@ def optional_artifacts_method(ints: list, num1: int, optional1: list = None,
     return result
 
 
-def variadic_input_method(ints: list, int_set: int, nums: int,
+def variadic_input_method(ints: list, int_set: set, nums: int,
```

Comment:
The correct type annotation is `int`. We don't currently have a way to communicate the view's container type back to the framework, so instead we register the view type of the elements, rather than the container.

```suggestion
def variadic_input_method(ints: list, int_set: int, nums: int,
```